code/game/machinery/Sleeper.dm

```diff
@@ -18,21 +18,34 @@
 	var/min_health = -25
 	var/list/available_chems
 	var/controls_inside = FALSE
-	var/list/possible_chems = list(
+	var/obj/item/reagent_containers/list/chem_stores
+	var/list/chem_store_names
+	var/selected_store = 0
+	var/list/possible_chem_stores = list(1, 2, 4, 6)

```

Comment:
This is the var that determines how many slots the sleeper has depending on the rating of its matter bin, feedback welcome.

Reply:
honestly, triple all of these at minimum.

Reply:
would it be so bad to set this to the amount of chems it had prior to this change